jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ContainerParameters.java

```diff
@@ -279,10 +287,14 @@ public String getCreationTime() {
     if (System.getProperty(PropertyNames.CONTAINER_CREATION_TIME) != null) {
       return System.getProperty(PropertyNames.CONTAINER_CREATION_TIME);
     }
-    return creationTime;
+    return creationTime.getOrNull();
```

Comment:
`getOrElse` with the default value of `EPOCH`, perhaps?

Reply:
This is now handled by `convention("EPOCH_PLUS_SECOND")` and `convention("EPOCH")` on the (now final) Property objects